plugins/woocommerce-admin/client/two-column-tasks/style.scss

```diff
@@ -229,10 +236,10 @@
 		position: absolute;
 		z-index: 0;
 		right: 6%;
-		top: 50%;
-		bottom: 50%;
-		margin-top: auto;
-		margin-bottom: auto;
+		// top: 50%;
+		// bottom: 50%;
+		// margin-top: auto;
+		// margin-bottom: auto;
```

Comment:
Do we want to leave these commented or just delete them?

Reply:
Ooops, I forgot I left that in.

Reply:
I removed it here: https://github.com/woocommerce/woocommerce/pull/32507/commits/8eb03eba4627e5d0ab05f82711e8dc48559b0dcc